apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/chronoLocalDateAssertions.kt

```diff
@@ -1,6 +1,6 @@
 @file:Suppress("JAVA_MODULE_DOES_NOT_READ_UNNAMED_MODULE" /* TODO remove once https://youtrack.jetbrains.com/issue/KT-35343 is fixed */)
 
-package ch.tutteli.atrium.api.fluent.en_GB.jdk8
+package ch.tutteli.atrium.api.fluent.en_GB
```

Comment:
don't change the package here, change it in the copied files (which are in the normal jvm module)

Reply:
this is the file in the jvm module folder, not change the package name here?

Reply:
ah... now I see, you moved it instead of copying it. So here it would be correct to adjust the package name but in this file we don't want to see the deprecations.

Reply:
@robstoll ?

Reply:
You need to two two separate things:
1. you need to copy (not move) every file from the jdk8 module to the normal jvm module
2. deprecate methods in the jdk8 extension, taking this file as example, every method in in apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/chronoLocalDateAssertions.kt => here you don't have to adjust the package name


It seems like you first deprecated methods in the jdk8 module and then moved/instead of copying the file to the jvm module. 
Does that make things clearer?

Reply:
got it, my mistake